YAFC/Windows/WelcomeScreen.cs

```diff
@@ -72,13 +72,16 @@ private enum EditType {
             errorScroll = new ScrollArea(20f, BuildError, collapsible: true);
             Create("Welcome to YAFC CE v" + YafcLib.version.ToString(3), 45, null);
 
-            if (cliProject != null) {
+            if (cliProject != null && !string.IsNullOrEmpty(cliProject.dataPath)) {
```

Comment:
`cliProject` is never `null` as you now set it with

> ProjectDefinition cliProject = CommandLineParser.Parse(args);

Reply:
It can be, since that's the default value for the parameter. If only `Program.cs` would instantiate a new `WelcomeScreen`, we'd be safe. But there is the option to return to the welcome screen from the main window ;-)
https://github.com/have-fun-was-taken/yafc-ce/blob/aa077c816c9ea80dec4dff612bf1855b54a339eb/YAFC/Windows/MainScreen.cs#L652

Reply:
A nice catch, forgot about this path to the welcome screen! :+1: